csv-bulk-signature/src/main.ts

```diff
@@ -90,30 +94,19 @@ async function generateSignatures(
     });
 
     await fs.promises.writeFile(
-      `./dist/signatures/${fullName.split(" ").join("")}.htm`,
+      `${SIGNATURES_PATH}/${fullName.split(" ").join("")}.htm`,
       html
     );
   });
 
-  console.log(
-    `${skippedRows.length ? "🙅‍♂️" : "👍"} Signatures:`,
-    processedRows.length,
-    `processed and`,
-    skippedRows.length,
-    `skipped`
-  );
+  console.log("👍 Signatures generated");
```

Comment:
move progress log out of generateSignatures...since it doesn't related to the generation of files